lib/tools/node-modules/node-modules-dest-copy.ts

```diff
@@ -46,8 +46,8 @@ export class TnsModulesCopy {
 			shelljs.mkdir("-p", targetDir);
 			shelljs.cp("-Rf", dependency.directory, targetDir);
 
-			//remove platform-specific files (processed separately by plugin services)
-			const targetPackageDir = path.join(targetDir, dependency.name);
+			// remove platform-specific files (processed separately by plugin services)
+			const targetPackageDir = path.join(isScoped ? this.outputRoot : targetDir, dependency.name);
```

Comment:
Ok, it looks like I'm still sleeping 😃 
Anyway, the whole method is confusing - we shouldn't have different logic for scoped dependencies, I suggest the following change:
```TypeScript
private copyDependencyDir(dependency: IDependencyData): void {
	if (dependency.depth === 0) {
		const targetPackageDir = path.join(this.outputRoot, dependency.name);

		shelljs.mkdir("-p", targetPackageDir);
		shelljs.cp("-Rf", dependency.directory, targetPackageDir);

		//remove platform-specific files (processed separately by plugin services)
		shelljs.rm("-rf", path.join(targetPackageDir, "platforms"));
	}
}
```

Also do you have an idea why are we checking only dependencies on level 0 ?